apps/i18n/javalab/en_us.json

```diff
@@ -122,6 +122,7 @@
   "onlyVisibleToYou": "teacher comments are only visible to you",
   "openedDate": "opened {date}",
   "programCompleted": "Program completed.",
+  "programEndedUnexpectedly": "Your program stopped unexpectedly. We may be out of capacity to run your program, or you may be having network issues. Please check your network connection and try again.",
```

Comment:
Do we want to mention/encourage verification in either of these cases (ex. become a verified teacher to get more capacity/run without restrictions or something)?

Reply:
Good question. This could show up for any user, and could also show up for network errors so I wanted to make it somewhat generic.

Reply:
Should we give them some sort of differentiator? i.e. if they're a verified user, it's likely a problem on their side. If they're an unverified user, it's likely a problem on our side?

Reply:
(^ asking, not suggesting)

Reply:
We could. I'd have to see how messy it is to get the verification status inside Java Lab. For verified users, we can remove the "out of capacity" message. For unverified, I think it could be either still, so we keep the same message?

Reply:
As far as I can tell, we don't currently have access to whether the user is a verified teacher or student of a verified teacher on the frontend. So it would be hard to customize this message without some plumbing through of that information.